templates/aframe-model-template/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
make sure to update these in `templates/index.json` (I could auto-update that file based on `package.json`s in the `templates/` directory, but I'll leave it for now, since we can add external packages that way easily)